CMakeLists.txt

```diff
@@ -191,6 +191,7 @@ if(WarpX_APP)
     target_sources(app PRIVATE Source/main.cpp)
 endif()
 
+#add_subdirectory(Source/ABLASTR)
```

Comment:
once we rebase/merge as #2263 is merged, we can add the extra include in the `# own headers` part above

Reply:
currently same root with intentional directory prefix. keep as is for now.